assets/prototype/application/generated/types.d.ts

```diff
@@ -0,0 +1,9310 @@
+export type Maybe<T> = T | null;
```

Comment:
@tn3rb I wasn't sure that this location is the best, but placed here because I saw in few instances `generated` folder being used.
I think you might have a better proposal on where we should keep this file.

Reply:
considering it contains logic pertaining to specific EE data model types like events, datetimes, tickets, etc, the it should go into the domains folder, so maybe:

```
assets/prototype/domain/shared/types.d.ts
```

Reply:
Moved, thx:
https://github.com/eventespresso/event-espresso-core/pull/2076/commits/c6e8116faa38bbf06c6ced4384f47815f6c1e346